WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -480,10 +434,6 @@ private extension NotificationDetailsViewController {
             fallthrough
         case .text:
             return NoteBlockTextTableViewCell.reuseIdentifier()
-        case .comment:
-            return NoteBlockCommentTableViewCell.reuseIdentifier()
-        case .actions:
-            return NoteBlockActionsTableViewCell.reuseIdentifier()
```

Comment:
I'm not sure what's going on here. Do we just not handle these two kinds of content, and let if fall to the `default` case and use the a default cell?

Reply:
It's a good point. This code is extremely hard to follow because it's parsing notification payload returned by the server. I'd suggest that instead of removing this code right away, let's add `wpAssertionFailure` in every code path we _think_ can be removed, and then remove the code only when we are confident it's dead.